docs/CONFIG.txt

```diff
@@ -1,9 +1,16 @@
 Configuring which patches are used
 ----------------------------------
 
-This repository has two sets of patches - one set for the keyboard
-changes and one for turning off the battery validation.  The battery
-patches are disabled by default, but are easy to enable.
+This repository has contains the following sets of patches for various ThinkPad models:
+
+* Patch to use the classical IBM keyboard instead of the newer Lenovo chiclet keyboard.
+* Turning off the authentic battery validation check.
+  The battery patches are disabled by default, but are easy to enable.
+* Swap Fn and CtlL keys so that the physical layout becomes CtlL, Fn (how it should be).
+  For older Thinkpad models like the xx00 series the EC can not be configured
+  to swap Fn and Ctrl. This setting would usually be exposed in the BIOS setup
+  or in nvram when using coreboot.
+  Enabled by default for models where this can not be configured.
```

Comment:
Possibly need to make it clear which patches are available for which types of laptop